lni.cls

```diff
@@ -305,19 +305,19 @@
 \newcommand{\authorrunning}[1]{%
    \fancyhead[LE]{\hspace{0.05cm}\oldsmall\thepage\hspace{5pt}#1}}
 \newcommand*{\email}[1]{\href{mailto:#1}{\urlstyle{same}\protect\nolinkurl{#1}}}
+\PassOptionsToPackage{bookmarks=false}{hyperref}
```

Comment:
Is this an intended change to disable the bookmarks?

Reply:
No, just a helpess attempt to prevent the hyperref option clash

Reply:
Maybe open an issue in the orcidlink package

Reply:
Perhaps bookmarks are already disabled anyhow, at least there is `\RequirePackage[bookmarks=false]{hyperref}` in lni.cls.
Therefore I am not so sure if that's really an issue with orcidlink, or whether LNI (as user of orcidlink) should avoid the option clash using `\PassOptionsToPackage{...}`.